ui/component/publishAdditionalOptions/index.js

```diff
@@ -1,9 +1,22 @@
 import { connect } from 'react-redux';
+import { createSelector } from 'reselect';
 import { selectPublishFormValues, doUpdatePublishForm } from 'lbry-redux';
+import { selectIsStillEditing } from 'redux/selectors/publish';
 import PublishPage from './view';
 
+const selectDefaultLanguage = createSelector(
```

Comment:
Please add this to `redux/selectors/settings`

Reply:
wouldn't it be better in `redux/selectors/publish` as it's specific to the publish page.

Reply:
@seanyesmunt

Reply:
Yes, you are correct.